lib/CSkillHandler.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * CSkillHandler.h, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+#pragma once
+
+#include "../lib/HeroBonus.h"
+#include "GameConstants.h"
+#include "IHandlerBase.h"
+
+class CSkillHandler;
+class CGHeroInstance;
+class CMap;
+class JsonSerializeFormat;

```

Comment:
It would be cool if you actually use Json serialization, but if not remove forward declaration.

Reply:
fixed.